source/adressering-av-undertegner.rst

```diff
@@ -1,78 +1,78 @@
-.. _adressering-av-undertegner:
+.. _addressing-of-signer:
 
-Adressering av undertegner
+Addressing of signer
 ***************************
-Undertegnere som skal signere i :ref:`portalflyt <signering-i-portalflyt>` kan adresseres på to forskjellige måter;
+Signers that are to sign in :ref:`portal flow <signing-in-portal-flow>` may be addressed in two different ways:
```

Comment:
Lurte på å kanskje endre dette til mer aktiv form, sånn?

```suggestion
You may address signers for :ref:`portal flow <signing-in-portal-flow>` in two different ways:
```

Synes liksom "Signers that are to sign..." hørtes litt rart ut, men mulig det er helt innafor altså.